modules/openy_traction_rec_import/src/Commands/OpenyTractionRecImportCommands.php

```diff
@@ -109,31 +97,25 @@ public function __construct(
    */
   public function import(array $options): bool {
     if (!$this->importer->isEnabled()) {
-      $this->logger()->notice(
-        dt('Traction Rec import is not enabled!')
-      );
+      $this->logger()->notice('Traction Rec import is not enabled!');
       return FALSE;
     }
 
     if (!$this->importer->acquireLock()) {
-      $this->logger()->notice(
-        dt('Can\'t run new import, another import process already in progress.')
-      );
+      $this->logger()->notice('Can\'t run new import, another import process already in progress.');
```

Comment:
In these messages we need to suggest Action item
```suggestion
      $this->logger()->notice('Can\'t run new import, another import process already in progress. Please retry after another import is finished.');
```